snap/snapcraft.yaml

```diff
@@ -0,0 +1,26 @@
+name: ktlint
+version: git
+summary: An anti-bikeshedding Kotlin linter with built-in formatter
+description: |
+   Kotlin linter in spirit of feross/standard (JavaScript) and gofmt (Go).
+
+confinement: devmode
+
+apps:
+  ktlint:
+    command: java -jar $SNAP/ktlint-0.0.0-SNAPSHOT-shaded.jar
+    plugs: [home]
+    environment:
+      PATH: $PATH:$SNAP/usr/lib/jvm/java-8-openjdk-amd64/bin
+
+parts:
+  ktlint:
+    source: .
+    plugin: nil
+    build-packages: [openjdk-8-jdk]
+    stage-packages: [openjdk-8-jre]
+    build: |
+      ./mvnw
```

Comment:
Is there a reason for this to be included?

Reply:
I think I just misread your ci scripts. Removed.